docs/Contributing.md

```diff
@@ -1,16 +1,15 @@
-Contributing
-============
+## Contributing Guide
 
-Thank you for considering contributing to Carp.
+Thank you for considering contributing to Carp!
 
 This document currently focuses on developer contributions but we welcome all kinds of contributions.
 
-
-Community
----------
+### Community
 The best place to start is to join the Carp Gitter channel over at
 [https://gitter.im/carp-lang/Carp](https://gitter.im/carp-lang/Carp).
 
+### Understanding the Compiler
+See [Hacking.md](hacking.md) for a gentle introduction to the Carp compiler internals.
 
 License
```

Comment:
Should we also change this heading, then?

Reply:
You mean Hacking.md becomes UnderstandingTheCompiler.md ?

Reply:
No, this was on the `License` heading which is still the old MD style.

Reply:
Now they all use two '#' (if that's what you meant)